my_app/views.py

```diff
@@ -4,4 +4,18 @@
 
 
 def home(request):
-    return render(request, template_name='base.html')
\ No newline at end of file
+    return render(request, template_name='base.html')
+
+
+def new_check(request):
+
+    news_content = request.POST.get('news_content')
+
+    # Consultar modelo e retornar veracidade
+    veracidade = 'FALSA'
```

Comment:
veracidade is in portuguese and the code is in english

Reply:
ok